ui/src/app/applications/components/resource-details/resource-details.tsx

```diff
@@ -120,7 +120,7 @@ export const ResourceDetails = (props: ResourceDetailsProps) => {
                 key: 'extension',
                 content: (
                     <div>
-                        <ExtensionComponent node={node} tree={tree} />
+                        <ExtensionComponent node={node} tree={tree} state={state} />
```

Comment:
Noticed another issue introduced in original PR. I think extension tab should be shown only of live resource is available. please change if at line 117 to 

```
if (ExtensionComponent && state) {
```